### PR TITLE
Add Yield Aggregators

### PR DESCRIPTION
**Pull request by melotik**

### CODE REVIEW

File: schemas/yield-aggregator/SCHEMA.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+| protocolName          | The name of the protocol.                                            | string |
+| chainId               | The standard chain id.                                               | number |
+| creationTimestamp     | The timestamp this pool was created.                                 | number |
+| creationBlockNumber   | The block number that this pool was created.                         | number |
+| underlyingTokenAddress| The contract address of the underlying token or deposited token.     | string |
+| underlyingTokenSymbol | The symbol of the underlying token token.                            | string |
+| receiptTokenAddress   | The contract address of the output or receipt token of this pool, if available.| string |
+| receiptTokenSymbol    | The symbol of the receipt token.                                     | string |
+| poolAddress           | The smart contract address of the pool.                              | string |
+| poolSymbol            | The symbol of the pool.                                              | string |
```
Comment (0xroll) [inline schemas/yield-aggregator/SCHEMA.md]: we need to adopt similar schema like the ones we have for DEXes 

additional columns :

underlyingTokenIndex
underlyingTokenDecimals
receiptTokenDecimals

File: schemas/yield-aggregator/SCHEMA.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| timestamp             | The timestamp of the snapshot.                                       | number |
+| protocolName          | The name of the protocol.                                            | string |
+| chainId               | The standard chain id.                                               | number |
+| poolAddress           | The address of the pool this user has a position in.                 | string |
+| userAddress           | The address of the user who has a position in the pool.              | string |
+| depositAmount         | The amount of the underlying token that the user deposited, decimal normalized.| number |
+| depositAmountUsd      | The deposit amount in USD.                                           | number |
+| feesUsd               | The total fees in USD for the given snapshot period.                 | number |
+| incentiveAmount       | The amount of incentives.                                            | number |
+| incentiveUsd          | The value of incentives in USD.                                      | number |
```
Comment (0xroll) [inline schemas/yield-aggregator/SCHEMA.md]: similar, add in token indexes to support multiple tokens

additional column : 

underlyingTokenIndex
underlyingTokenAddress

change of column names : 
depositAmount -> underlyingTokenAmount
depositAmountUsd -> underlyingTokenAmountUsd

File: schemas/yield-aggregator/SCHEMA.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Pool Snapshot
+| Property              | Description                                                          | Type   |
+|-----------------------|----------------------------------------------------------------------|--------|
+| timestamp             | The timestamp of the snapshot.                                       | number |
+| protocolName          | The name of the protocol.                                            | string |
+| chainId               | The standard chain id.                                               | number |
+| poolAddress           | The address of the pool.                                             | string |
+| depositAmount         | The amount of underlying token deposited in this pool, decimal normalized.| number |
+| depositAmountUsd      | The total value locked in this pool of the deposited token in USD.   | number |
```
Comment (0xroll) [inline schemas/yield-aggregator/SCHEMA.md]: additional column :

underlyingTokenIndex
underlyingTokenAddress

change of column names :
depositAmount -> underlyingTokenAmount
depositAmountUsd -> underlyingTokenAmountUsd

File: schemas/yield-aggregator/schema.json [hunk 4]
```diff
@@ -0,0 +1,180 @@
+{
+    "schema": "Yield Aggregator (Leverage, Gambling, RWA, ALM, Liquidity/LP, index)",
+    "description": "Standard table definitions for yield aggregator protocols (can include, Leverage, Gambling, RWA, ALM, Liquidity/LP, index).",
+    "version": "1.0.0-alpha",
+    "tables": [
+      {
+        "label": "Pools",
+        "tableName": "misc_pools",
```
Comment (0xroll) [inline schemas/yield-aggregator/schema.json]: should they be classified as `misc`?

if we are grouping them all under `misc` then we might need a category in the pools schema

Comment (melotik) [reply schemas/yield-aggregator/schema.json]: I haven't changed this from our internal table names defined in the FE.

It is a TODO item to make better names, but it hasn't been a priority yet

**Review by 0xroll — APPROVED** [review]

**Pull request merged**